### PR TITLE
Add inactive instance status

### PR DESCRIPTION
**Pull request by MykolaSolyanko**

### CODE REVIEW

File: api/cloudprotocol/cloudprotocol.go [hunk 1]
```diff
@@ -95,8 +95,9 @@ const (
 
 // Service instance states.
 const (
-	InstanceStateActive = "active"
-	InstanceStateFailed = "failed"
+	InstanceStateActive   = "active"
```
Comment (al1img) [inline api/cloudprotocol/cloudprotocol.go:98]: Also add activating state here

**Review by mlohvynenko — APPROVED** [review]

File: api/cloudprotocol/cloudprotocol.go (hunk 1 above)
Comment (MykolaSolyanko) [reply api/cloudprotocol/cloudprotocol.go]: Done

File: api/cloudprotocol/cloudprotocol.go [hunk 2]
```diff
@@ -95,9 +95,10 @@ const (
 
 // Service instance states.
 const (
-	InstanceStateActive   = "active"
-	InstanceStateFailed   = "failed"
-	InstanceStateInactive = "inactive"
+	InstanceStateActive     = "active"
```
Comment (al1img) [inline api/cloudprotocol/cloudprotocol.go:98]: Put it in logical order: activating, active, inactive, failed

**Review by al1img — APPROVED** [review]

File: api/cloudprotocol/cloudprotocol.go (hunk 2 above)
Comment (MykolaSolyanko) [reply api/cloudprotocol/cloudprotocol.go:99]: Done

**Review by al1img — APPROVED** [review]

**Pull request merged**